apps/client/src/app/auth-content/auth-content.tsx

```diff
@@ -33,6 +27,21 @@ const AuthProvider: React.FC = ({ children }) => {
     return () => unlisten();
   }, []);
 
+  useEffect(() => {
```

Comment:
The route for signup will be POST /api/auth/register and get the token from the 'Authentication' Header and get the first/lastname from the body. So overall just set the idToken to:
`fetch(URL, {
  method: 'POST',
  headers: {
   // ...
   'Authorization' : idToken,
  },
  body: JSON.stingify({ firstname, lastname }),
});`

With the idToken I am validating the user from firebase, and on the register I am creating a User in DB.